examples/jsm/controls/TransformControls.d.ts

```diff
@@ -28,7 +28,7 @@ export class TransformControls extends Object3D {
   isTransformControls: boolean;
   visible: boolean;
 
-  attach(object: Object3D): void;
+  attach(object: Object3D): this;
```

Comment:
Arguably it should, but the method doesn't appear to return `this` today:

https://github.com/mrdoob/three.js/blob/93e72ba7b24958ddb0652bd33171edd14ed2d693/examples/js/controls/TransformControls.js#L140-L155

Reply:
please look at dev branch 
at TransformControls in the jsm directory (not js)
it is different :/

Reply:
i will provide a link

Reply:
Ok. The real issue is that Transform Controls extends Object3d
and Object3d returns "this" in that method:
https://github.com/mrdoob/three.js/blob/f86fe1a8ec00c56fd5ffe88a6af3357e30b78fa5/src/core/Object3D.d.ts#L293

Reply:
This method in `Object3D` was added lately ,see https://github.com/mrdoob/three.js/pull/16528.

Reply:
It would be bad if this issue blocks the other change for `ColladaLoader`. How about making two PRs? This one focuses just on `TransformControls` and a new one for `ColladaLoader`(which can be immediately merged).